odysseus/halow_setup/install/nrc-led

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+######################
+# amount of time to wait in between loops
+SLEEP_TIME=1
+#interface to check for connectivity, wlan0 for default newracom configuration
+INTERFACE_NAME=wlan0
+# the URL or IP to ping, if ping returns good yellow light is lit, if it doesnt then it is not lit
+YELLOW_LED_CHECK_URL="8.8.8.8"
+# path to NRC cli app
+CLI_APP="/home/pi/nrc_pkg/script/cli_app"
+# maximum time to wait for a yellow light ping, minimun 1 second
+MAX_PING_TIME=1
+######################
+
+
+
+# bit flipper for blinking function
+oscillate=0
+
+# run led code only after module has been loaded, shouldn't be needed if started via systemd
+if [[ $(lsmod | grep "nrc") -eq 0 ]];
+then
+  echo "nrc kernel module not present!"
+  exit
+fi
+
+    
+# set the correct pin directions, must run before a write
+$CLI_APP gpio direction 3 1
+$CLI_APP gpio direction 2 1
+
+# pin 3 = red led (RX)
+# pin 2 = yellow red (TX)
+
+# a cleanup script run upon any exit (like ctrl^c), to turn the lights off and not leave them on and unresponsive.  Should run before module is unloaded.
+trap cleanup EXIT
+cleanup() {
+    $CLI_APP gpio write 3 0
+    $CLI_APP gpio write 2 0
+}
+
+while true;
+do
+    sleep "$SLEEP_TIME"s
```

Comment:
Solid implementation, we might want to increase the sleep time more tho since this isn't the most high priority thing

Reply:
Alright will do.  Maybe in the future we could switch it to mosquitto diagnostic of sort or argos as we talked about.